pkg/trie/triedb/in_memory_to_triedb_migration_test.go

```diff
@@ -8,20 +8,19 @@ import (
 
 	"github.com/ChainSafe/gossamer/internal/database"
 	"github.com/ChainSafe/gossamer/pkg/trie"
-	"github.com/ChainSafe/gossamer/pkg/trie/inmemory"
+	inmemory_trie "github.com/ChainSafe/gossamer/pkg/trie/inmemory"
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 )
 
-func newTestDB(t *testing.T) database.Table {
+func newTestDB(t assert.TestingT) database.Table {
 	db, err := database.NewPebble("", true)
-	require.NoError(t, err)
+	assert.NoError(t, err)
```

Comment:
why this change?

Reply:
To remove the `require` dependency and use `assert` for every test